securedrop/source_templates/base.html

```diff
@@ -28,16 +28,22 @@
         {% endblock %}
 
         <div class="panel selected">
+          {% if g.show_offline_message %}
+          <h1>{{ gettext("We're sorry, our SecureDrop is currently offline.") }}</h1>
+          <p>{{ gettext("Please try again later. Check our website for more information.") }}</p>
+          {% else %}
+
           {% if 'logged_in' in session %}
-            <a href="{{ url_for('main.logout') }}" class="btn pull-right" id="logout">{{ gettext('LOG OUT') }}</a>
+          <a href="{{ url_for('main.logout') }}" class="btn pull-right" id="logout">{{ gettext('LOG OUT') }}</a>
```

Comment:
looks like the changes on lines 37, 41, 44 and 46 are not strictly required here and make the diff more complicated to read

Reply:
ah, i see. i was trying to mimic the code style i saw in other areas of the code. it looked like other areas of the code try to keep the indents of the html correct rather than the template code, but if you want i can revert to make it easier for you to review.

Reply:
easier to review but also preserves a cleaner history, making it easier to revert changes in the future if/when this code is no longer be necessary (i.e. 2.0)